src/frontend/src/icons/CrateDB/CrateDB.jsx

```diff
@@ -0,0 +1,300 @@
+              id="path35"/>
+        </clipPath>
+        <clipPath
+            clipPathUnits="userSpaceOnUse"
+            id="cp13">
+          <path
+              d="m818.32 23.53h-31.98v39.98h-31.98v27.99h31.98v55.96c0.29 9.72 4.4 18.92 11.44 25.63 7.03 6.71 16.42 10.37 26.14 10.2h34.38v-27.83h-39.98v-63.96h39.97v-27.99h-39.97z"
+              id="path38"/>
+        </clipPath>
+        <clipPath
+            clipPathUnits="userSpaceOnUse"
+            id="cp14">
+          <path
+              d="m882.29 63.51h132.21v120.49h-132.21z"
+              id="path41"/>
+        </clipPath>
+        <clipPath
+            clipPathUnits="userSpaceOnUse"
+            id="cp15">
+          <path
+              d="m1002.5 75.22c7.5 7.5 11.71 17.67 11.71 28.27v31.98h-99.94v21.99h91.95v25.99h-83.96c-10.6 0-20.78-4.21-28.27-11.71-7.49-7.5-11.7-17.67-11.7-28.27v-39.98c0-10.61 4.2-20.78 11.7-28.27 7.5-7.5 17.67-11.71 28.27-11.71h51.97c10.6 0 20.78 4.2 28.27 11.71zm-16.2 38.26v-12.07c-0.02-3.16-1.3-6.2-3.54-8.43-2.25-2.23-5.28-3.48-8.45-3.48h-47.97c-3.18 0-6.23 1.27-8.48 3.51-2.25 2.25-3.52 5.3-3.52 8.48v11.99z"
+              id="path44"/>
+        </clipPath>
+        <clipPath
+            clipPathUnits="userSpaceOnUse"
+            id="cp16">
+          <path
+              d="m1046.19 23.53h167.81v160.47h-167.81z"
+              id="path47"/>
+        </clipPath>
+        <clipPath
+            clipPathUnits="userSpaceOnUse"
+            id="cp17">
+          <path
+              d="m1197.71 39.92c10.5 10.5 16.39 24.74 16.39 39.58v47.97c0 14.85-5.9 29.08-16.39 39.58-10.5 10.5-24.73 16.4-39.58 16.4h-111.94v-159.92h111.94c14.85 0 29.08 5.89 39.58 16.39zm-15.59 39.59c0-6.37-2.51-12.47-7.02-16.97-4.51-4.49-10.61-7.02-16.97-7.02h-79.95v95.94h79.94c6.37 0 12.47-2.52 16.97-7.02 4.5-4.51 7.03-10.6 7.03-16.97v-47.97z"
+              id="path50"/>
+        </clipPath>
+        <clipPath
+            clipPathUnits="userSpaceOnUse"
+            id="cp18">
+          <path
+              d="m1246.08 23.93h175.92v160.07h-175.92z"
+              id="path53"/>
+        </clipPath>
+        <clipPath
+            clipPathUnits="userSpaceOnUse"
+            id="cp19">
+          <path
+              d="m1420.36 123.85c2.46 9.48 1.95 19.48-1.44 28.66-3.39 9.18-9.52 17.1-17.54 22.7-8.03 5.61-17.58 8.62-27.37 8.64h-127.93v-159.92h119.94c8.78 0.02 17.37 2.43 24.87 7 7.5 4.56 13.61 11.1 17.65 18.89 4.04 7.8 5.87 16.55 5.27 25.3-0.59 8.76-3.57 17.19-8.61 24.37 7.41 6.38 12.71 14.88 15.16 24.36zm-142.29-36.36h87.95c4.24 0 8.3-1.68 11.3-4.68 3.01-2.99 4.7-7.06 4.7-11.31 0-4.24-1.7-8.3-4.7-11.3-2.99-3-7.06-4.68-11.3-4.68h-87.95zm107.2 57.93c2.89-3.31 4.48-7.56 4.48-11.96 0-4.4-1.59-8.65-4.48-11.95-2.89-3.31-6.9-5.45-11.26-6.04h-95.94v35.98h95.94c4.36-0.58 8.37-2.73 11.26-6.03z"
+              id="path56"/>
+        </clipPath>
+        <style>
+          <![CDATA[
+          .a { fill: #19c0ea }
+          .b { fill: #fff }
+          ]]>
+        </style>
+      </defs>
```

Comment:
This possibly fixes the SVG/CSS flaw: Following advises in [^1], it slots the CSS style definitiions into the `<defs>` section, and wraps them into a `<![CDATA[ ... ]]>` block.

[^1]: https://developer.mozilla.org/en-US/docs/Web/SVG/Tutorial/Fills_and_Strokes#using_css